bin/yaml/cpp.yaml

```diff
@@ -695,6 +703,7 @@ compilers:
           - patmat-trunk
           - p1061-trunk
           - reflection-trunk
+          - amd-stg-open
```

Comment:
Ah! For parity with the other builds (plus some pretty unfortunate, but in the process of being fixed, reasons) we should probably call this `rocm-trunk` and not `amg-stg-open`.

Reply:
I'll merge and fix once I have got the build working. I think this is 99% my misunderstandings of your project branch structure :) I hope `rocm-trunk` === `amd-stg-open` and they aren't something completely different.